### DISCUSSION TITLE
Status display layout

### DISCUSSION THREAD

**Discussion by Jkh62** [jgyates/genmon#1078] [Ideas]
Is it possible now (or for a future mod) to change the layout of the display gauges on the status page and save the view or profile ?
I like putting the L1 & L2 next to each other and group the pi WIFI and temp gauges next to each other , graph on the bottom , etc.

Learning more as I go and watching it run and tinker with it.

**Comment by jgyates** [comment]
I will add this to the list of requested features. FYI, the grouping that you like is the default, however the current code will automatically adjust the rows to fit on the screen so the layout is also dictated by the width of the monitor (and browser window).

**Reply by jgyates** [reply]
The ability to save the gauge layout is now part of the 2.0 update.

**Comment by Jkh62** [comment]
Fantastic work.
Looking forward to it's release.
